src/components/appLayout/Overview.tsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import HeaderContent from './HeaderContent';
+import OverviewLayout from './OverviewLayout';
+import SideMenu from './SideMenu';
+
+const Overview: React.FC = () => {
+  const [isMenuFolded, setIsMenuFolded] = useState(false);
+
+  const sideMenu = <SideMenu></SideMenu>;
```

Comment:
```suggestion
  const sideMenu = <SideMenu />;
```

Be consistent in how you close tags if they have no children